metagpt/provider/openai_api.py

```diff
@@ -323,3 +329,40 @@ async def amoderation(self, content: Union[str, list[str]]):
     async def _amoderation(self, content: Union[str, list[str]]):
         rsp = await self.llm.Moderation.acreate(input=content)
         return rsp
+
+
+class CustomedGPTAPI(OpenAIGPTAPI):
```

Comment:
Should inherited from BaseGPTAPI, because custom api can be wenxingyiyan or spark api.
But if you means that it's the models which keep the same openai api format, it's fine.